docroot/sites/policy.uiowa.edu/modules/policy_core/src/Controller/PDFContentController.php

```diff
@@ -166,27 +166,30 @@ public static function batchFinished($success, $results, $operations): void {
       return;
     }
 
-    $print_styles = '';
-
-    // Include local fonts.
-    $path_resolver = \Drupal::service('extension.path.resolver');
-    $path = $path_resolver->getPath('module', 'policy_core');
-    $font_css = DRUPAL_ROOT . '/' . $path . '/css/pdf.css';
-    $print_styles .= '<style>' . file_get_contents($font_css) . '</style>';
+    $fonts = '<link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100..900;1,100..900&family=Zilla+Slab:ital,wght@0,300;0,400;0,500;0,600;0,700;1,300;1,400;1,500;1,600;1,700&display=swap" rel="stylesheet">';
```

Comment:
@bspeare I copied this straight from Google Fonts. I doubt we need all of these weights right?